### ISSUE TITLE
Can i add a run_exports?

### ISSUE THREAD

**Issue by hmaarrfk** [conda-forge/conda-forge-repodata-patches-feedstock#224]
I've been trying to check if anybody patched in a run_exports.

Is that possible to do? Buliding (and uploading) pytorch is rather hard.

**Comment by jakirkham** [comment]
@chenghlee do you know if this is possible?

**Comment by chrisburr** [comment]
It's not possible as conda-build takes `run_exports` from the local package metadata during the build. IIRC `run_exports` doesn't exist in the channel's `repodata.json` at all.

**Comment by jakirkham** [comment]
Yeah I tried looking in our repodata for `run_exports` info, but didn't find anything. Though didn't know if this might be something that could/would be added in the future

**Comment by jakirkham** [comment]
That said, Mark, it should be possible to fix existing packages built with `pytorch` that have the problematic `run_exports`. Perhaps that is just as useful in practice?

**Comment by hmaarrfk** [comment]
Yea, the immediate problem is that compiling pytorch takes time. and I don't entirely understand the debugging process for repodata patches. I'll get to it at some point

**Comment by jakirkham** [comment]
Sure just trying to provide an option that may be less painful. That said haven't looked at what the `run_exports` issue with `pytorch` is

**Comment by hmaarrfk** [comment]
The recipe was split and I didn't add it to the right output....

**Comment by jakirkham** [comment]
Got it. Yeah there's probably nothing we can do with the repodata then.

**Comment by jakirkham** [comment]
This is being discussed upstream in issue: https://github.com/conda/ceps/pull/75
